src/main/java/uk/gov/justice/digital/delius/service/AppointmentService.java

```diff
@@ -58,21 +48,34 @@ public AppointmentService(ContactTypeRepository contactTypeRepository,
     public AppointmentCreateResponse createAppointment(String crn, Long sentenceId, AppointmentCreateRequest request) {
         this.assertAppointmentType(request.getContactType());
 
-        NewContact newContact = makeNewContact(crn, sentenceId, request);
-        ContactDto contactDto = deliusApiClient.createNewContract(newContact);
+        final var newContact = makeNewContact(crn, sentenceId, request);
+        final var contactDto = deliusApiClient.createNewContact(newContact);
 
         return new AppointmentCreateResponse(contactDto.getId());
     }
 
     public AppointmentCreateResponse createAppointment(String crn, Long sentenceId, String contextName, ContextlessAppointmentCreateRequest contextualRequest) {
 
-        IntegrationContext context = getContext(contextName);
-        Requirement requirement = requirementService.getRequirement(crn, sentenceId, context.getRequirementRehabilitationActivityType());
-        AppointmentCreateRequest request = appointmentOf(contextualRequest, requirement.getRequirementId(), context);
+        final var context = getContext(contextName);
+        final var requirement = requirementService.getRequirement(crn, sentenceId, context.getRequirementRehabilitationActivityType());
+        final var request = appointmentOf(contextualRequest, requirement.getRequirementId(), context);
 
         return createAppointment(crn, sentenceId, request);
     }
 
+    public AppointmentUpdateResponse patchAppointment(String crn, Long appointmentId, JsonPatch jsonPatch) {
+
+        final var contactDto = deliusApiClient.patchContact(appointmentId, jsonPatch);
+        return new AppointmentUpdateResponse(contactDto.getId());
+    }
```

Comment:
I think it would make sense here to assert that the contact is an appointment type. Otherwise this endpoint could be abused.

Reply:
Do this require two checks:
1. The underlying contact has the correct appointment type
2. The patch is not changing the appointment type